components/crowdfunding-redesign/Fundraiser.tsx

```diff
@@ -190,46 +195,51 @@ const ContentOverview = ({ content }) => {
         About
       </Link>
       {headingTexts.map(heading => (
-        <Link href="#" key={heading} className={linkClasses()}>
+        <Link href="#" key={heading} className={linkClasses()} onClick={triggerPrototypeToast}>
           {heading}
         </Link>
       ))}
     </div>
   );
 };
 
-export default function Fundraiser(props) {
-  const account = props.account;
-
-  const title = account.parent ? account.name : `Support ${account.name}`;
+export default function Fundraiser({ account }) {
+  const fundraiser = getDefaultFundraiserValues(account);
   const mainAccount = account.parent || account;
   return (
     <React.Fragment>
       <div className="border-b bg-background">
-        <div className="mx-auto flex h-16 max-w-screen-xl items-center justify-between">
+        <div className="mx-auto flex h-16 max-w-screen-xl items-center justify-between px-6">
           <Link href={`/preview/${account.parent?.slug || account.slug}`}>
             <Avatar className="" collective={account.parent || account} />
           </Link>
         </div>
       </div>
       <div className="">
-        <div className="mx-auto max-w-screen-xl space-y-8 py-12">
+        <div className="mx-auto max-w-screen-xl space-y-8 px-6 py-12">
           <div className="space-y-4 text-center">
-            <h1 className="text-balance text-4xl font-semibold">{title}</h1>
-            <p className="text-lg">{account.description}</p>
+            <h1 className="text-balance text-4xl font-semibold">{fundraiser.name}</h1>
+            <p className="text-lg">{fundraiser.description}</p>
           </div>
 
           <div className="flex grow gap-8">
             {/* main */}
             <div className="w-full max-w-[900px] grow space-y-4">
-              <div className="relative h-96 w-full overflow-hidden rounded-lg bg-primary/20">
-                {account.backgroundImageUrl && (
-                  <img src={account.backgroundImageUrl} alt="background" className="h-full w-full object-cover" />
-                )}
+              <div className="relative aspect-video w-full overflow-hidden rounded-lg bg-primary/20">
+                {fundraiser.cover?.type === 'IMAGE' ? (
+                  // eslint-disable-next-line jsx-a11y/alt-text
+                  <img src={fundraiser.cover.url} className="h-full w-full object-cover" />
```

Comment:
I tend to use `alt=""` when alt text is not really required, it plays better with accessibility inspector tools as they don't show them as "Missing alt text". Not a blocker for me to use one or the other.

Reply:
Good tip! Changed to `alt=""`